### ISSUE TITLE
Ubuntu CI Failing

### ISSUE THREAD

**Issue by Robadob** [FLAMEGPU/FLAMEGPU2_dev#426]
The Ubuntu Github action image has updated, bringing with it CMake verison 3.19.

Googletest doesn't support this yet (), which is calling all ubuntu CI runs to fail.

Won't fix it ourselves just yet, hopefully Googletest will be updated in the very near future.

GTest issue: https://github.com/google/googletest/issues/3040
GTest candidate PR: https://github.com/google/googletest/pull/3094 (month old without being merged at time of writing).

**Comment by ptheywood** [comment]
Installing a pinned version (3.18.4) is a possible (temporary) fix:

```
- name: Install Specific Cmake version via Pip.
      run: |
        python3 -m pip install --user cmake==3.18.4
        echo "$(python3 -m site --user-base)/bin" >> $GITHUB_PATH

**Comment by ptheywood** [comment]
#427 will be a short term fix for this, until a gtest update is available, and cmake > 3.18 can be used.
